R/pointr.R

```diff
@@ -0,0 +1,57 @@
+#' Download a dataset from sharepoint
```

Comment:
We talked about a parse function - it would be useful to have a parse function that takes any URL and tries to find these pieces.  Another ticket though I think

Reply:
Added a ticket for that https://mrc-ide.myjetbrains.com/youtrack/issue/RESIDE-119

Will look at picking that up asap

Reply:
thanks, this is great